app/views/pages/personal_dashboard.html.erb

```diff
@@ -1 +1,60 @@
-<h1>Personal dashboard</h1>
+<div class="container">
+  <h1>Personal dashboard</h1>
+
+  <!-- list users details -->
+  <div class="personal-details">
+
+    <h2>Account</h2>
+    <img src="" alt="avatar" class="rounded-circle">
+    <%= current_user.first_name %>
+    <%= current_user.last_name %>
+    <%= current_user.address %>
+    <%= current_user.email %>
+    <button type="button" class="btn"></button>
+    <%= link_to 'edit', edit_user_registration_path %>
+
+    <button type="button" class="btn"></button>
+    <%= link_to 'delete', cancel_user_registration_path %>
+  </div>
+
+      <!-- list users userplans -->
+  <div class="user-plan">
+
+    <h2>Plans</h2>
+    <ul>
+      <% @plans.each do |user_plan| %>
+      <li>
+        <%= user_plan.order.subscription.name %>
+        Left usage:
+        <%= user_plan.left_usage %>
+      </li>
+      <% end %>
+    </ul>
+
+    <!-- user can create new user plan -->
+    <%= link_to 'Create New User Plan', new_user_plan_path %>
```

Comment:
a user cannot create a new user plan, a new user plan is created by buying a new subscription. could u instead index here the diffrent subscription models so that we can initiate the payment process?